README.md

```diff
@@ -43,69 +43,95 @@ remotes::install_github("mrc-ide/frogger", upgrade = FALSE)
+| artcd4elig_idx     | art.idx_cd4_elig                         | The index of the CD4 count at which people are eligible for ART by time step                                                                                          |
+| art_alloc_mxweight | art.initiation_mortality_weight          | Weighting for extent that expected mortality guides ART uptake                                                                                                        |
+| art_dropout        | art.dropout                              | ART dropout rate at each time step                                                                                                                                    |
 
 ### Outputs
 
-| Leapfrog         | Frogger            | Details                                                  |
-|------------------|--------------------|----------------------------------------------------------|
-| totpop1          | total_population   | Projected total population                               |
-| hivpop1          | hiv_population     | Projected HIV population                                 |
-| infections       | infections         |                                                          |
-| hivstrat_adult   | hiv_strat_adult    |                                                          |
-| artstrat_adult   | art_strat_adult    |                                                          |
-| births           | births             | Projected number of births                               |
-| natdeaths        | natural_deaths     | Projected number of natural deaths                       |
-| natdeaths_hivpop | hiv_natural_deaths | Projected number of natural deaths within HIV population |
-| hivdeaths        | hiv_deaths         |                                                          |
-| aidsdeaths_noart | aids_deaths_no_art |                                                          |
-| aidsdeaths_art   | aids_deaths_art    |                                                          |
-| artinit          | art_initiation     |                                                          |
+Discussion: naming conventions
+
+* Distinguish events vs. counts
+* Distinguish stratification of array (single-year population pXX vs. HIV population hXX
+
+| Leapfrog         | Frogger              | Details                                                                  |
+|------------------|----------------------|--------------------------------------------------------------------------|
+| totpop1          | p_total_population   | Projected total population                                               |
+| hivpop1          | p_hiv_population     | Projected HIV population                                                 |
```

Comment:
I continue to prefer `p_totpop` and `p_hivpop` for conciseness and clarity, but won't insist on it and happy to relent.

Reply:
Updated to `p_total_pop` and `p_hiv_pop`, I want to keep the naming consistent so with a `_` between the two words. That ok?